src/account-settings/components/PasswordReset.jsx

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
-import { FormattedMessage } from 'react-intl';
+import { FormattedMessage, injectIntl, intlShape } from 'react-intl';
```

Comment:
```suggestion
import { FormattedMessage } from 'react-intl';
import { injectIntl, intlShape } from '@edx/frontend-i18n';
```
Since we shim injectIntl to prevent errors we should import injectIntl and shape from @edx/frontend-i18n

Reply:
Oh, right!! Thanks.  I auto-imported.